azure.yaml

```diff
@@ -0,0 +1,12 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/Azure/azure-dev/main/schemas/alpha/azure.yaml.json
+
+name: gs-spring-boot-for-azure
+metadata:
+  template: gs-spring-boot-for-azure@0.0.1-beta
+services:
+  demo:
+    project: ./complete/
+    host: springapp
+    language: java
+    spring:
```

Comment:
It's odd that we have to specify a deployment name here when it is "default".  What happens when we don't specify?  It should use an intelligent default...help the user work less.

Reply:
thanks for pointing it out, will be fixed by pr https://github.com/spring-guides/gs-spring-boot-for-azure/pull/27